### ISSUE TITLE
Pod CIDR

### ISSUE THREAD

**Issue by boocup** [kubernetes-incubator/kube-aws#1513]
I am trying to modify the Pod CIDR used by nodes. I have uncommented the "podCIDR:" in cluster.yaml, but the Pod CIDR shown in the node page of the dashboard doesn't reflect my change. Is there more that needs to be done? 

My Steps: I have done Kube-aws destroy. Then after modifying the field in cluster.yml ran kube-aws up. Should I expect the change to be reflected? Apologies if I am missing something.

BTW- I am trying to get more IPs available so I can have more pods per node.

**Comment by davidmccormick** [comment]
Hi 

You can change the `podCIDR` using `kube-aws` but it's a disruptive process and one that is not well supported by kubernetes.   You need to change the `podCIDR` in the `cluster.yaml` (you can remove it and take the default but I would recommend you specify your own) and then roll your cluster.  When the first apiserver comes up it will see that the podCIDR has changed and delete all the nodes and pods in the cluster (and any services if you changed the serviceCIDR).  However, if your new CIDR range *contains* the original one it won't have to do this.  You will then need to wait until all apiservers and nodes have rolled before the new podCIDR which will affect both pods and nodes is completely active in your cluster and service is restored.

**Comment by davidmccormick** [comment]
btw - what version of kube-aws are you using? also can you post the logs from the `install-kube-system` systemd service from the first apiserver that comes up?

**Comment by boocup** [comment]
Hi. Thanks for the quick reply!  I am using kube-aws version v0.12.0. 

Yes, I did try modifying the Pod CIDR via uncommenting and change the podCIDR: setting in the cluster.yaml file. I ran kube-aws destroy, modified the yaml file and then ran kube-aws up. But the reported node cidr in the dashboard did not reflect the change. It still had 10.2.1.0/24 and I requested 16.

If these are the correct steps to make it work, I'll try it again. This is a test cluster so any disruption is not an issue.

Thanks again for getting back to me.
